documentation/general/general_setup.md

```diff
@@ -125,6 +125,7 @@ ones you might see used throughout the documentation:
 - [`httpie`](https://httpie.io/docs/cli) is a command line HTTP client. Like
   cURL but with a lot of conveniences and easier to remember. Try `http --help`
   for info.
+- ['psql'] (https://www.postgresql.org/docs/current/app-psql.html) is an advanced Object-Relational database management system (DBMS)
```

Comment:
The documentation for `psql` is incorrect. Also it might be nice to add `pgcli` to this list.

```suggestion
- ['psql'] (https://www.postgresql.org/docs/current/app-psql.html) is the PostgreSQL client
```